### PR TITLE
RSDK-7839 enable disabling of replanning during motion calls

### PR DESCRIPTION
**Pull request by raybjork**

### CODE REVIEW

File: services/motion/builtin/builtin_test.go [hunk 1]
```diff
@@ -639,8 +645,8 @@ func TestStoppableMoveFunctions(t *testing.T) {
 			motionCfg := motion.MotionConfiguration{
 				PlanDeviationMM:       10000,
 				LinearMPerSec:         10,
-				PositionPollingFreqHz: 4,
-				ObstaclePollingFreqHz: 1,
+				PositionPollingFreqHz: &defaultPositionPollingHz,
```
Comment (biotinker) [inline services/motion/builtin/builtin_test.go:648]: If these are defaults, don't they not need to be specified?

Comment (raybjork) [reply services/motion/builtin/builtin_test.go:648]: I might be able to remove them.  I was just doing 1:1 swaps

File: services/motion/motion_test.go [hunk 2]
```diff
@@ -828,29 +828,6 @@ func TestMoveOnGlobeReq(t *testing.T) {
 	name := "somename"
 	dst := geo.NewPoint(1, 2)
 
-	t.Run("String()", func(t *testing.T) {
```
Comment (raybjork) [inline services/motion/motion_test.go:831]: This whole test seems fragile and not worth it IMO

Comment (biotinker) [reply services/motion/motion_test.go:831]: Agree

**Review by biotinker — APPROVED** [review]
LGTM

**Comment by biotinker** [comment]
Note that the extended motion tests need to be updated, as this changes typing on public methods

**Pull request merged**